deisctl/cmd/platform.go

```diff
@@ -11,12 +11,18 @@ import (
 
 // InstallPlatform loads all components' definitions from local unit files.
 // After InstallPlatform, all components will be available for StartPlatform.
-func InstallPlatform(b backend.Backend, checkKeys func() error) error {
+func InstallPlatform(b backend.Backend, checkKeys func() error, stateless bool) error {
 
 	if err := checkKeys(); err != nil {
 		return err
 	}
 
+	if stateless {
+		fmt.Println("Warning: With a stateless control plane, `deis logs` will be unavailable.")
+		fmt.Println("Additionally, components will need to be configured to use external persistent stores.")
+		fmt.Println("See the official Deis documentation for details on running a stateless control plane.")
```

Comment:
@carmstrong 

> again, I think having controller ensure the log directory exists on startup is the way to go here

As far as I can tell (and I've double-checked this a few times) it already works that way.  Sanity check for me... have you seen it behave to the contrary?